packages/apps/shopify-app-remix/docs/MIGRATION_V3.md

```diff
@@ -32,3 +32,84 @@ export async function loader({request}: LoaderFunctionArgs) {
   return null;
 }
 ```
+
+## Root import path deprecation
+
+Apps can no longer import server-side functions using the following statements:
+
+```diff
+import '@shopify/shopify-app-remix/adapters/node';
++ import {shopifyApp} from '@shopify/shopify-app-remix';
+```
+
+With the addition of React component to this package, we've separated the exports between server and react code, as in:
+
+```diff
+import '@shopify/shopify-app-remix/server/adapters/node';
++ import {shopifyApp} from '@shopify/shopify-app-remix/server';
+- import {shopifyApp} from '@shopify/shopify-app-remix';
+import {AppProvider} from '@shopify/shopify-app-remix/react';
+```
+
+## Use the AppProvider component
```

Comment:
Also moved this here. We discussed having it in the template but we currently don't have any migration guides there. Would we still like to document it there?